app/views/account/pages/create.js.erb

```diff
@@ -0,0 +1,2 @@
+$("#content-body").html("<%= escape_javascript(render @pages) %>");
```

Comment:
a new page is visible after adding?
for example you have only 5 pages in list
and you adding new
would you see 6 pages?

Reply:
with current scope it won't be visible, also according to the collection if user creates new it'll show all company pages
e.g.:
if there are 100 pages and user creates new one system will show all 101 pages

this is definitely not correct